lib/chewy/type/import.rb

```diff
@@ -84,19 +84,26 @@ def bulk_body(action_objects, indexed_objects = nil)
           end
         end
 
+        # Returns the value of the identifier for an object
```

Comment:
I considered just removing this method entirely and inlining the contents in the two places where it is used, but decided that I felt the functional encapsulation still made sense and kept the separate method.